.gitignore

```diff
@@ -186,3 +186,4 @@ dmypy.json
 
 # Visual Studio Code Files
 .vscode
+.devcontainer
```

Comment:
Usually repos want to "ship" the devcontainers already in the `.devcontainer`. Are we leaving this one in an example folder because we do not want to commit it yet?